packages/preact-router/src/Transitioner.tsx

```
@@ -0,0 +1,128 @@
+import { useEffect, useState, useRef } from 'preact/hooks'
```

ESLint

[error] Member 'useRef' of the import declaration should be sorted alphabetically.

(sort-imports)